pom.xml

```diff
@@ -138,6 +142,12 @@
         </dependency>
 
         <dependency>
+            <groupId>javax.mail</groupId>
```

Comment:
The exception is coming from the method I mentioned above. But the method definition is same as some other previously defined Response methods in `GtfsFeed.java` and `GtfsRtFeed.java` for ajax requests. There is nothing newly added in the Response types in the above method. But still, the exception is seen.